src/quo2/components/icons/svg.cljs

```diff
@@ -34,8 +34,74 @@
        "M9.15142 9.99998L7.07566 12.0757L7.9242 12.9243L9.99994 10.8485L12.0757 12.9242L12.9242 12.0757L10.8485 9.99998L12.9242 7.92421L12.0757 7.07568L9.99994 9.15145L7.92421 7.07572L7.07568 7.92425L9.15142 9.99998Z"
        :fill color-2}]]))
 
+(defn- dropdown-20
+  [{:keys [color color-2] :as props}]
+  (let [color   (or color colors/neutral-100)
+        color-2 (or color-2 colors/white)]
+    [container props
+     [svg/path
+      {:d
+       "M3 10C3 6.13401 6.13401 3 10 3C13.866 3 17 6.13401 17 10C17 13.866 13.866 17 10 17C6.13401 17 3 13.866 3 10Z"
+       :fill color}]
+     [svg/path
+      {:d
+       "M7 8.5L10 11.5L13 8.5"
+       :stroke color-2
+       :stroke-width 1.2}]]))
+
+(defn- pullup-20
+  [{:keys [color color-2] :as props}]
+  (let [color   (or color colors/neutral-100)
+        color-2 (or color-2 colors/white)]
+    [container props
+     [svg/path
+      {:d
+       "M3 10C3 6.13401 6.13401 3 10 3C13.866 3 17 6.13401 17 10C17 13.866 13.866 17 10 17C6.13401 17 3 13.866 3 10Z"
+       :fill color}]
+     [svg/path
+      {:d
+       "M7 11.5L10 8.5L13 11.5"
+       :stroke color-2
+       :stroke-width 1.2}]]))
+
+(defn- dropdown-12
+  [{:keys [color color-2] :as props}]
+  (let [color   (or color colors/neutral-100)
+        color-2 (or color-2 colors/white)]
+    [container props
+     [svg/circle
+      {:cx   6
+       :cy   6
+       :r    6
+       :fill color}]
+     [svg/path
+      {:d
+       "M3.5 5L6 7.5L8.5 5"
+       :stroke color-2
+       :stroke-width 1.1}]]))
+
+(defn- pullup-12
+  [{:keys [color color-2] :as props}]
+  (let [color   (or color colors/neutral-100)
+        color-2 (or color-2 colors/white)]
+    [container props
+     [svg/circle
+      {:cx   6
+       :cy   6
+       :r    6
+       :fill color}]
+     [svg/path
+      {:d
+       "M3.5 7L6 4.5L8.5 7"
+       :stroke color-2
+       :stroke-width 1.1}]]))
```

Comment:
Interesting approach.

Why are we generating these svgs dynamically instead of adding a png file and coloring it as we do with our icons?

About those
`(let [color (or color colors/something))`

I don't like too much the idea of shadowing variables (we are receiving `color` and then creating the let value again), some times it's handy, but I always try to avoid it, since it feels like we are working with mutable structures (`color` becomes a var that changes its value across the function's body).

We can destructure with `:or` and give a default value to `color`, which I think is what you want:
```clojure
[{:keys [color color-2]
  :or {color colors/something
       color-2 colors/something-else}]
```

wdyt?


And about vars name, `color` and `color-2` are not the best names IMO, too generic, and we know what they are being used for. What about `:fill-color` and `stroke-color` respectively?

Reply:
@ulisesmac - We use SVG if the icon has two colors. We can NOT use PNG as the `tint-color` (the colouring method we use) will apply to the whole image.

Regarding the shadowed var, I can update it to use the `:or` to prevent shadowing. 👍 Thanks! 

> And about vars name, color and color-2 are not the best names IMO, too generic, and we know what they are being used for. What about :fill-color and stroke-color respectively?

I don't think we can do that as each icon SVG may have `fill` or `stroke`. For example, the existing `clear` icon SVG uses fill but the `dropdown` and `pullup` icons use stroke. 

Additionally, we expose these SVGs via the Icon component. That's why we used the generic name `color` and `color-2`.